### ISSUE TITLE
Support react-native-vector-icons 4.1.0

### ISSUE THREAD

**Issue by valerybodak** [react-native-training/react-native-elements#356]
I have installed react-native-elements today but I have the same issue:
https://github.com/react-native-training/react-native-elements/issues/140
![untitled-1](https://cloud.githubusercontent.com/assets/26433088/25743933/c25f5df8-319f-11e7-8bfe-93c1ac1dbe65.png)

Could you please resolve this issue.

Thank you.

**Comment by binoy14** [comment]
@valerybodak you need to install the react-native-vector-icons separately take a look here https://react-native-training.github.io/react-native-elements/Installation/default_installation/

**Comment by bennygenel** [comment]
@binoy14 react-native-vector-icons current version is 4.1.0. it gives unmet peer dependency error even if you installed it separately

```
+-- react-native-elements@0.11.2 
| +-- react-native-side-menu@0.20.1 
| `-- react-native-tab-navigator@0.3.3 
|   `-- immutable@3.8.1 
`-- UNMET PEER DEPENDENCY react-native-vector-icons@4.1.0

npm WARN react-native-elements@0.11.2 requires a peer of react-native-vector-icons@~4.0.0 but none was installed.
```

package.json

```
{
   ...
  "dependencies": {
      ...,
      "react-native-vector-icons": "^4.1.0",
   }
}
```

**Comment by Joe312341** [comment]
Had the same thing, when I fixed `react-native-icons` to version 4.0.0 it stopped complaining

**Comment by binoy14** [comment]
I see what the issue is. We will try to fix it asap. In the mean time as a work around if you do `npm install react-native-vector-icons@4.0.0` that error should go away.

**Comment by fmoraw** [comment]
The point is you can't build for IOs if react is > 40.

**Comment by Joe312341** [comment]
if u fix it to version 4.0.0 you can build for IOS (im using react vers 44)

**Comment by binoy14** [comment]
Should be fixed as part of #354

**Comment by binoy14** [comment]
This has been fixed and will be released `0.12.0`
